helix-view/src/document.rs

```diff
@@ -121,6 +124,7 @@ pub struct Document {
     pub(crate) id: DocumentId,
     text: Rope,
     selections: HashMap<ViewId, Selection>,
+    pub view_data: HashMap<ViewId, ViewData>,
```

Comment:
Looks like this is only used within `document`

```suggestion
    view_data: HashMap<ViewId, ViewData>,
```

Reply:
Fixed